### ISSUE TITLE
v1 parser not resolving common parameters that use relative references

### ISSUE THREAD

**Issue by cmuchinsky** [swagger-api/swagger-parser#1087]
Using the v1 parser, top level parameters that use relative references are not being resolved. Take for example parsing the following swagger fragment:
```
...
parameters:
  param2:
    $ref: "./parameters/params.yaml#/param2"
...
```
Actual results:
```
...
parameters:
  param2:
    $ref: "./parameters/params.yaml#/param2"
...
```
Expected results:
```
...
parameters:
  param2:
    name: "param2"
    in: "header"
    required: false
    type: "string"
...
```

**Comment by gracekarina** [comment]
closed by #1088

**Comment by cmuchinsky** [comment]
@gracekarina any thoughts on when a v1 release with this fix might be coming?
